packages/features/bookings/lib/handleNewBooking/global-booking-limits.test.ts

```diff
@@ -545,6 +555,305 @@ describe(
+            },
+          },
+        });
+
+        // this is the fourth booking of this week for user 101, limit reached
+        await expect(
+          async () => await handleNewBooking({ bookingData: mockBookingData2 })
+        ).rejects.toThrowError("booking_limit_reached");
+      });
+      test(`Booking limits per month`, async ({}) => {
+        await createBookingScenario(
+          getScenarioData({
+            eventTypes: [
+              {
+                id: 1,
+                slotInterval: eventLength,
+                length: eventLength,
+                schedulingType: null,
+                userId: 101,
+              },
+              {
+                id: 2,
+                slotInterval: eventLength,
+                length: eventLength,
+                schedulingType: null,
+                userId: 101,
+              },
+            ],
+            bookings: [
+              {
+                eventTypeId: 1,
+                userId: 101,
+                status: BookingStatus.ACCEPTED,
+                startTime: `2024-08-03T03:30:00.000Z`,
+                endTime: `2024-08-03T04:00:00.000Z`,
+              },
+              {
+                eventTypeId: 2,
+                userId: 101,
+                status: BookingStatus.ACCEPTED,
+                startTime: `2024-08-22T03:30:00.000Z`,
+                endTime: `2024-08-22T04:00:00.000Z`,
+              },
+            ],
+            organizer: organizerWithBookingLimits({ PER_MONTH: 3 }),
+          })
+        );
+
+        const mockBookingData1 = getMockRequestDataForBooking({
+          data: {
+            start: `2024-08-29T04:30:00.000Z`,
+            end: `2024-08-29T05:00:00.000Z`,
+            eventTypeId: 1,
+            responses: {
+              email: booker.email,
+              name: booker.name,
+              location: { optionValue: "", value: "New York" },
+            },
+          },
+        });
+
+        const createdBooking = await handleNewBooking({ bookingData: mockBookingData1 });
+
+        expect(createdBooking.responses).toEqual(
+          expect.objectContaining({
+            email: booker.email,
+            name: booker.name,
+          })
+        );
+
+        const mockBookingData2 = getMockRequestDataForBooking({
+          data: {
+            start: `2024-08-25T04:00:00.000Z`,
+            end: `2024-08-25T04:30:00.000Z`,
+            eventTypeId: 2,
+            responses: {
+              email: booker.email,
+              name: booker.name,
+              location: { optionValue: "", value: "New York" },
+            },
+          },
+        });
+
+        // this is the fourth booking of this month for user 101, limit reached
+        await expect(
+          async () => await handleNewBooking({ bookingData: mockBookingData2 })
+        ).rejects.toThrowError("booking_limit_reached");
+      });
+      test(`Booking limits per year`, async ({}) => {
```

Comment:
@CarinaWolli We do have tests for booking limits per year

Reply:
This test is for `handleNewBooking`, which is working as expected and throws the error when the limit is reached. The actual bug is in availability loading, it shouldn't show open slots if the limit is already reached